lang/javascript/javascript.py

```diff
@@ -248,6 +248,16 @@ def code_insert_function(text: str, selection: str):
         actions.user.paste(text)
         actions.edit.left()
 
+    def code_default_function(text: str):
```

Comment:
Ok I see that you have moved this one into Javascript, so now I'm confused.  Won't that mean that there's no way of outputting a function without modifiers in Typescript, like you suggested [earlier](https://github.com/knausj85/knausj_talon/pull/879/files/16c12cb8cf5f372df760d3af0b679832268ac2d6#r895266125)?

Reply:
It works as you'd expect; `function helloWorld` output by `funky hello world` in both cases. The reason is typescript.talon enables the user.javascript tag which layers in all the stuff in javascript.py.

The implementation of code_default_function prior to this PR called code_private_function. In typescript this output `private function helloWorld`, and in javascript `function helloWorld`. After this PR code_default_function (for both .js and .ts) explicitly outputs `function helloWorld`, it doesn't delegate to another action.

I moved the definition up to javascript because I thought you were objecting to where I'd put it semantically (even if the behaviour was correct). The argument there would be something like: 'typescript is a specialised version of javascript, and so common behaviour should be in the javascript level'.